frontend/passkeys.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', function() {
+    'use strict';
+
+    const registerPasskeyButtons = document.querySelectorAll('#register-passkey');
```

Comment:
```suggestion
    const registerPasskeyButton = document.querySelectorAll('#register-passkey');
```

Reply:
I have refactored this so use a 'querySelector' method and optional chaining.